benchmark/buffers/buffer-concat.js

```diff
@@ -2,20 +2,20 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  pieces: [1, 4, 16],
+  pieces: [4, 16],
   pieceSize: [1, 16, 256],
   withTotalLength: [0, 1],
-  n: [1024]
+  n: [8e5]
```

Comment:
Running `concat` on a single entry is somewhat redundant and should not be the average case. That's why I removed it. The iterations are now clearer in the result and lowered all together.